src/wmnds/components/disruption-indicator/medium/_medium.njk

```diff
@@ -1,45 +1,44 @@
 {% macro wmndsDisruptionIndicatorMedium(params) %}
 {% from "wmnds/components/icon/_icon.njk" import wmndsIcon %}
 
-{% set strongText = params.strongText if params.strongText else "Indicator text" %} {# set button text #}
 {% set iconRight = params.iconRight if params.iconRight %} {# set icon right classes #}
 {% set iconLeft = params.iconLeft if params.iconLeft %} {# set svg left reference #}
-{% set text = params.text if params.text else 'X15' %} {# set text, or default to X15 #}
+{% set _content = params.contentText if params.contentText else 'X15' %} {# set text, or default to X15 #}
 
-{% set class = ' ' + params.classModifiers if params.classModifiers %}
-{% set iconClassModifier = ' wmnds-disruption-indicator-medium--with-icon' if ( params.iconLeft or params.iconRight ) %}
+{% set classes = ' ' + params.classes if params.classes %}
+{% set _iconClassModifier = ' wmnds-disruption-indicator-medium--with-icon' if ( params.iconLeft or params.iconRight ) %}
```

Comment:
It may be worth adding a param for narrow and make it a boolean. From the component page I couldn't figure out how to use the narrow variant without looking at this code and realising I needed an icon left or right